docs/how-to/design-tokens-support.rst

```diff
@@ -0,0 +1,189 @@
+- **Modify**: Optional value that helps to apply a specific token modification.
+
+Use the ``source`` attribute to map the tokens in Paragon and create the theme files. Also, it will help you to replace the values in scss files if you have custom variables (see below).
+
+You can check `Paragon tokens <https://github.com/openedx/paragon/tree/alpha/tokens>`_ to know the folder and token structure, and how to work with modifiers.
+
+
+Build the tokens and generate the CSS variables
+===============================================
+
+To build the tokens you can use Paragon CLI. You can check its options `here <https://github.com/openedx/paragon?tab=readme-ov-file#paragon-cli>`_. 
+For using the help command run:
+
+.. code-block:: bash
+    
+    npm run pgn -- <name of the command you want to see more information>
+
+#. Install Paragon as a dev dependency
+
+.. code-block:: bash
+    
+    npm ci
+
+#. Once the tokens have been created. Go to the ``package.json``, there is a script template:
+
+.. code-block:: json
+    
+    {
+      "scripts": {
+        "build-tokens": "paragon build-tokens --source ./tokens/ --build-dir  <destination_path> -t <theme_variant> -t <theme_variant>"
+      }
+    }
+    
+Replace the destination with the desired path and specify the theme variant to be compiled, then run the command. 
+You can explore the command options using Paragon help for ``build-tokens``.
+
+.. code-block:: bash
+    
+    npm run build-tokens
+
+# Create the CSS distribution files modifying the template script in ``package.json``:
+
+.. code-block:: json
+    
+    {
+      "scripts": {
+         "build-dist-files": "rm -rf dist && mkdir dist && paragon build-scss --corePath <core_scss_file> --themesPath <themes_css_files> --source",
+      }
+    }
+
+Where ``<themes_css_files>`` is the path created when you run the build-tokens command and ``<core_scss_file>`` is the path to the scss core file, 
+by default **./paragon/core.scss**. Then run the command:
+
+.. code-block:: bash
+    
+    npm run build-dist-files
+
+You can explore the command options using Paragon help for ``build-scss``. Finally, this command should create 3 diffent files (.css, .map, .min.css) for the
+core and each of the available themes; also, the command creates a ``theme-urls.json`` (see below for more details).
+
+#. Publish the package. 
+
+#. Once it is installed in the application use the Paragon CLI with the ``replace-variables`` command to use your custom tokens.
+
+
+The ``theme-urls.json`` file
+=============================
+
+It is recommended to create the `theme-urls.json` if you are working with runtime theming for use the installed package as a fallback, or if want to use 
+``ParagonWebpackPlugin`` to preload the token URLs during the application build time.
+
+The file must be in the ``dist`` folder and should looks like this:
+
+.. code-block:: json
+    
+    {
+		  "themeUrls": {

```

Comment:
Could you please ensure that each code example in this PR has the same indentation consistently?